mne/tests/test_source_estimate.py

```diff
@@ -37,6 +38,8 @@
 fname_src_3 = op.join(data_path, 'subjects', 'sample', 'bem',
                       'sample-oct-4-src.fif')
 fname_stc = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-meg')
+fname_vec_stc = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc-meg-vector.h5')
```

Comment:
still an issue. Actually do you even need this file? Can't you just make an estimate, save and load, and check that the round-trip is correct?

For validating that the initial computation is correct (if that is a concern), you can check that `stc_vec.magnitude().data` is numerically equivalent to `stc_nonvec.data`